db/migrate/20190506075749_add_recorded_at_for_retroactive_data_entry.rb

```diff
@@ -0,0 +1,6 @@
+class AddRecordedAtForRetroactiveDataEntry < ActiveRecord::Migration[5.1]
```

Comment:
This migration won't run since 20190510143922_make_caller_phone_number_in_call_logs_not_nullable has already run. We should update the file name.

Reply:
updated